dali/operators/image/distortion/jpeg_compression_distortion_op_gpu.cu

```diff
@@ -38,17 +37,63 @@ class JpegCompressionDistortionGPU : public JpegCompressionDistortion<GPUBackend
   std::vector<int> quality_;
 };
 
+template <int ndim>
+TensorListShape<ndim - 1> unfold_dim(const TensorListShape<ndim> &shapes) {
```

Comment:
```suggestion
TensorListShape<ndim - 1> unfold_first_dim(const TensorListShape<ndim> &shapes) {
```
?

Reply:
Was `unfold_first_dim` for a second, but later thought to myself that unfold always starts from first dimention. Can change that.

Reply:
How about `unfold_outer_dim`?

Reply:
:+1: to `unfold_outer_dim`

```suggestion
TensorListShape<ndim != DynamicDimensions ? ndim - 1 : DynamicDimensions>
unfold_outer_dims(const TensorListShape<ndim> &shapes) {
```

Reply:
@mzient  `unfold_outer_dim` or `unfold_outer_dims`?